src/api/schema.py

```diff
@@ -78,3 +78,36 @@ class ChatResponse(BaseChatResponse):
 class ChatStreamResponse(BaseChatResponse):
     choices: list[ChoiceDelta]
     object: Literal["chat.completion.chunk"] = "chat.completion.chunk"
+
+
+class EmbeddingsRequest(BaseModel):
+    input: str | list[str]
+    model: str
+    # Cohere Embed
+    input_type: Literal["search_document", "search_query", "classification", "clustering"] | None = None
+    truncate: Literal["NONE", "LEFT", "RIGHT"] | None = None
+    # Titan Embeddings
+    embedding_config: dict | None = None
+
+
+class BaseEmbeddingsResponse(BaseModel):
+    created: int = Field(default_factory=lambda: int(time.time()))
+    model: str
+    system_fingerprint: str = "fp_e97c09dd4e26"
```

Comment:
You can remove this. There is no system_fingerprint in embedding response: https://platform.openai.com/docs/api-reference/embeddings/create